src/server/game/Entities/Object/Object.cpp

```diff
@@ -1484,11 +1484,12 @@ float WorldObject::GetGridActivationRange() const
 {
     if (isActiveObject())
     {
-        if (ToPlayer()->GetCinematicMgr()->IsOnCinematic())
-            return DEFAULT_VISIBILITY_INSTANCE;
-        return GetMap()->GetVisibilityRange() * 5;
```

Comment:
This commit reverted what SnapperRy dubbed 'temporary workaround', Is it 100% sure that it's intended or a side-effect of conflict solving?